public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Facebook — Connect with friends and the world around you</title>
```

Comment:
Also remove all the extra text from this file, you can remove all the inner text, only the tags can be left here